client/app/scripts/ng-sir-trevor-blocks.js

```diff
@@ -438,12 +439,15 @@ define([
                         contenteditable: true,
                         placeholder: that.descriptionPlaceholder
                     }).html(data.caption));
-                    
+
                     this.$editor.append($('<div>', {
                         name: 'credit',
                         class: 'st-image-block',
                         contenteditable: true,
-                        placeholder: that.authorPlaceholder
+                        placeholder: that.authorPlaceholder,
+                        oninput:
+                        'if(this.innerText.length > 300) { alert("' + that.limitedText + '"); this.innerText = this.innerText.substring(0, 300); this.style="border:1px solid red"}' +
```

Comment:
That is the first thing I have tried of course. If I define a function there( when appending the input to the editor) or pass a defined one, it just gets called once on render and then nothing.
I found a way to add that functionality in proper way after it has been rendered